### PR TITLE
Added info about manually disabling and enabling cron jobs

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Documented process to manually disable and enable cron jobs in Cloud environments using the new ece-tools CLI commands `cron:disable` and `cron:enable`.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/cloud/configure-setup-cron-jobs.html
- https://devdocs.magento.com/guides/v2.3/cloud/release-notes/ece-tools-release-notes.html

whatsnew
- Added instructions to manage cron processing manually using the ece-tools CLI commands `cron:disable` and `cron:enable`  to the [Set up cron jobs](https://https://devdocs.magento.com/guides/v2.3/cloud/configure-setup-cron-jobs.html#disable-cron-jobs") topic in the _Cloud Guide_.

### CODE REVIEW

File: src/guides/v2.2/cloud/release-notes/ece-tools-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,544 @@
...
+
+-  {:.fix}Fixes and improvements
+
+## v2002.1.0
+
+-  {:.new}Decoupled the Docker package from the `{{ site.data.var.ct }}` package to maintain code quality and provide independent releases. Updates and fixes related to `{{ site.data.var.cd }}` are managed from the [`magento-cloud-docker`](https://github.com/magento/magento-cloud-docker) GitHub repository.<!--MAGECLOUD-3986-->
+
+-  {:.new}Extended the functionality of the WARM_UP_PAGES environment variable to support cache preloading for specific product pages. See the expanded definition in the [post-deploy variables]({{ page.baseurl }}/cloud/env/variables-post-deploy.html#warm_up_pages) topic.<!--MAGECLOUD-4444-->
+
+-  {:.new}**New cron command**–You can now manually disable and enable cron jobs in your Magento Commerce Cloud environment using the `cron:disable` and `cron:enable` command. See [Disable cron jobs]({{ page.baseurl }}/cloud/configure/setup-cron-jobs.html#disable-cron-jobs)
```
Comment (meker12) [inline src/guides/v2.2/cloud/release-notes/ece-tools-release-notes.md]:
```suggestion
-  {:.new}**New cron command**–You can now manually manage cron processing in your Magento Commerce Cloud environment using the `cron:disable` and `cron:enable` commands. Use the disable command to stop all active cron processes and disable all Magento cron jobs.  Use the enable command to re-enable cron jobs when you are ready.  See [Disable cron jobs]({{ page.baseurl }}/cloud/configure/setup-cron-jobs.html#disable-cron-jobs)
```

File: src/guides/v2.2/cloud/configure/setup-cron-jobs.md [hunk 2]
```diff
@@ -154,14 +154,37 @@ To add custom crons:
...
 To add, remove, or update a custom cron job, change the configuration in the `crons` section of the `.magento.app.yaml` file for the Integration environment. Then, test the updates in the Integration environment before pushing the changes to the Production and Staging environments.
 
+## Disable cron jobs
+
+In some cases you might want to manually disable cron jobs before you complete maintenance tasks like reindexing or cleaning the cache to prevent performance issues. You can use the ece-tools CLI command `cron:disable` to disable all Magento cron jobs and stop any active cron processes.
+
+{:.procedure}
+To disable cron jobs when completing maintenance tasks:
+
+1. [SSH to your environment]({{ page.baseurl }}/cloud/env/environments-ssh.html#ssh).
```
Comment (hguthrie) [inline src/guides/v2.2/cloud/configure/setup-cron-jobs.md]: Use SSH to log in to ....

File: src/guides/v2.2/cloud/configure/setup-cron-jobs.md [hunk 3]
```diff
@@ -154,14 +154,37 @@ To add custom crons:
...
+
+1. Disable cron jobs and stop active cron processes.
+
+   ```shell
+   ./vendor/bin/ece-tools cron:disable
+   ```
+
+1. Complete the required task.
```
Comment (hguthrie) [inline src/guides/v2.2/cloud/configure/setup-cron-jobs.md]: This seems vague. The step is to complete the task, but is a task presented after running cron:disable? Or is the task waiting for something to complete? If it is waiting, then would be better to warn that stopping active process may take time.

File: src/guides/v2.2/cloud/configure/setup-cron-jobs.md [hunk 4]
```diff
@@ -154,14 +154,37 @@ To add custom crons:
...
+
+1. After the task completes, re-enable the cron jobs.
+
+   ```shell
+   ./vendor/bin/ece-tools cron:enable
+   ```
+
 ## Troubleshooting cron jobs
 
 Magento has updated the {{site.data.var.ece}} package to optimize cron processing on the {{site.data.var.ece}} platform and to fix cron-related issues. If you are having problems with cron processing, make sure that your project is using the most current version of the ece-tools package. See [Upgrades and patches]({{ page.baseurl }}/cloud/project/project-upgrade-parent.html).
```
Comment (hguthrie) [inline src/guides/v2.2/cloud/configure/setup-cron-jobs.md]:
```suggestion
Magento has updated the {{site.data.var.ece}} package to optimize cron processing on the {{site.data.var.ece}} platform and to fix cron-related issues. If you encounter problems with cron processing, make sure that your project is using the most current version of the ece-tools package. See [Upgrades and patches]({{ page.baseurl }}/cloud/project/project-upgrade-parent.html).
```

File: src/guides/v2.2/cloud/configure/setup-cron-jobs.md (hunk 2 above)
Comment (meker12) [inline src/guides/v2.2/cloud/configure/setup-cron-jobs.md]:
```suggestion
1. Use [SSH to log in to your environment]({{ page.baseurl }}/cloud/env/environments-ssh.html#ssh).
```

File: src/guides/v2.2/cloud/configure/setup-cron-jobs.md [hunk 5]
```diff
@@ -154,14 +154,35 @@ To add custom crons:
...
 ## Update custom cron jobs {#update}
 
 To add, remove, or update a custom cron job, change the configuration in the `crons` section of the `.magento.app.yaml` file for the Integration environment. Then, test the updates in the Integration environment before pushing the changes to the Production and Staging environments.
 
+## Disable cron jobs
+
+In some cases you might want to manually disable cron jobs before you complete maintenance tasks like reindexing or cleaning the cache to prevent performance issues. You can use the ece-tools CLI command `cron:disable` to disable all Magento cron jobs and stop any active cron processes.
+
+{:.procedure}
+To disable cron jobs:
```
Comment (meker12) [inline src/guides/v2.2/cloud/configure/setup-cron-jobs.md]:
```suggestion
To disable cron jobs:
```

File: src/guides/v2.2/cloud/configure/setup-cron-jobs.md [hunk 6]
```diff
@@ -154,17 +154,38 @@ To add custom crons:
...
+1. Disable cron jobs and stop active cron processes.
+
+   ```shell
+   ./vendor/bin/ece-tools cron:disable
+   ```
+   
+After you complete the maintenance task, ensure that you enable the cron jobs again.
+
+```shell
+./vendor/bin/ece-tools cron:disable
```
Comment (meker12) [inline src/guides/v2.2/cloud/configure/setup-cron-jobs.md]:
```suggestion
./vendor/bin/ece-tools cron:enable
```

File: src/guides/v2.2/cloud/configure/setup-cron-jobs.md [hunk 7]
```diff
@@ -154,17 +154,38 @@ To add custom crons:
...
+
+1. Use [SSH to log in to your environment]({{ page.baseurl }}/cloud/env/environments-ssh.html#ssh).
+
+1. Disable cron jobs and stop active cron processes.
+
+   ```shell
+   ./vendor/bin/ece-tools cron:disable
+   ```
+   
+After you complete the maintenance task, ensure that you enable the cron jobs again.
```
Comment (meker12) [inline src/guides/v2.2/cloud/configure/setup-cron-jobs.md]:
```suggestion
After you complete any required maintenance tasks, ensure that you enable the cron jobs again.
```

File: src/guides/v2.2/cloud/configure/setup-cron-jobs.md [hunk 8]
```diff
@@ -154,17 +154,38 @@ To add custom crons:
...
+After you complete any required maintenance tasks, ensure that you enable the cron jobs again.
+
+```shell
+./vendor/bin/ece-tools cron:enable
+```
+
 ## Troubleshooting cron jobs
 
-Magento has updated the {{site.data.var.ece}} package to optimize cron processing on the {{site.data.var.ece}} platform and to fix cron-related issues. If you are having problems with cron processing, make sure that your project is using the most current version of the ece-tools package. See [Upgrades and patches]({{ page.baseurl }}/cloud/project/project-upgrade-parent.html).
+Magento has updated the {{site.data.var.ece}} package to optimize cron processing on the {{site.data.var.ece}} platform and to fix cron-related issues. If you encounter problems with cron processing, make sure that your project is using the most current version of the ece-tools package. See [Upgrades and patches]({{ page.baseurl }}/cloud/project/project-upgrade-parent.html).
```
Comment (meker12) [inline src/guides/v2.2/cloud/configure/setup-cron-jobs.md]:
```suggestion
Magento has updated the {{site.data.var.ece}} package to optimize cron processing on the {{site.data.var.ece}} platform and to fix cron-related issues. If you encounter problems with cron processing, make sure that your project is using the most current version of the {{ site.data.var.ct }} package. See [Upgrades and patches]({{ page.baseurl }}/cloud/project/project-upgrade-parent.html).
```

**Comment by meker12** [comment]
> Just need a clarification in steps. Also, don't use SSH as a verb.
> Is this the PR with the necessary release notes topic? I thought this was only for crons update.

In order to add the release note, I merged the new release notes content into this branch.  I still have an integration branch for the release notes waiting for PO review.

File: src/guides/v2.2/cloud/configure/setup-cron-jobs.md [hunk 9]
```diff
@@ -154,17 +154,38 @@ To add custom crons:
...
+
+1. Use [SSH to log in to your environment]({{ page.baseurl }}/cloud/env/environments-ssh.html#ssh).
+
+1. Disable cron jobs and stop active cron processes.
+
+   ```shell
+   ./vendor/bin/ece-tools cron:disable
+   ```
+   
+After you complete any required maintenance tasks, ensure that you enable the cron jobs again.
```
Comment (hguthrie) [inline src/guides/v2.2/cloud/configure/setup-cron-jobs.md]: It makes sense that this is a step, too, now that it is more clear.

Comment (meker12) [inline src/guides/v2.2/cloud/configure/setup-cron-jobs.md]:
```suggestion
1. After you complete any required maintenance tasks, ensure that you enable the cron jobs again.
```

**Pull request merged**